Shared/Samples/Add feature layer with time offset/AddFeatureLayerWithTimeOffsetView.swift

```diff
@@ -0,0 +1,127 @@
+        let noOffsetLayer = FeatureLayer(featureTable: ServiceFeatureTable(url: .hurricanesService))
+        
+        // Applies a blue dot renderer to distinguish hurricanes without offset.
+        noOffsetLayer.renderer = SimpleRenderer(symbol: SimpleMarkerSymbol(style: .circle, color: .blue, size: 10))
+        
+        // Makes the offset hurricanes feature layer.
+        let withOffsetLayer = FeatureLayer(featureTable: ServiceFeatureTable(url: .hurricanesService))
+        
+        // Applies a red dot renderer to distinguish the hurricanes with an offset.
+        withOffsetLayer.renderer = SimpleRenderer(symbol: SimpleMarkerSymbol(style: .circle, color: .red, size: 10))
+        
+        // Sets the time offset on the layer.
+        withOffsetLayer.timeOffset = TimeValue(duration: 10.0, unit: .days)
+        
+        // Adds the layers to the map.
+        map.addOperationalLayers([noOffsetLayer, withOffsetLayer])
+        
+        // Sets the initial viewpoint of the map.
+        map.initialViewpoint = Viewpoint(latitude: 45.0, longitude: -45.0, scale: 9e7)
+        
+        return map
+    }()
+    
+    /// The time extent of the map view.
+    @State private var timeExtent: TimeExtent? = TimeExtent(
+        startDate: .august4th2000,
+        endDate: Calendar.current.date(byAdding: .day, value: 10, to: .august4th2000)
+    )
+    
+    /// The value of the slider.
+    @State private var sliderValue = 0.0
+    
+    /// The maximum value of the slider.
+    private let sliderMaxValue = Double(
+        Calendar.current.dateComponents([.day], from: Date.august4th2000, to: Date.october22nd2000).day!
+    )
+    
+    /// The format style used to display the extent dates.
+    private let dateFormat = Date.FormatStyle(date: .numeric, time: .omitted)
+    
+    var body: some View {
+        MapView(map: map, timeExtent: $timeExtent)
+            .overlay(alignment: .topTrailing) {
+                VStack(alignment: .leading) {
+                    Text("Hurricane Data Offset:")
+                    LabeledContent {
+                        Text("10 days")
+                    } label: {
+                        Text("Red").foregroundStyle(.red)
+                    }
+                    LabeledContent {
+                        Text("No offset")
+                    } label: {
+                        Text("Blue").foregroundStyle(.blue)
+                    }
+                }
+                .fixedSize()
+                .padding()
+                .background(.thinMaterial)
+                .clipShape(.rect(cornerRadius: 10))
+                .shadow(radius: 50)
+                .padding()
+            }
+            .toolbar {
+                ToolbarItem(placement: .bottomBar) {
+                    VStack {
+                        Slider(value: $sliderValue, in: 0...sliderMaxValue, step: 1)
+                            .padding(.top)
+                            .padding(.horizontal)
```

Comment:
```suggestion
                            .padding([.top, .horizontal])
```

These can be combined. (I personally think it looks better without top padding since the padding pushes the date pretty close to the home bar, but that's just me.)

Reply:
I removed the top padding. On the canvas it looked like it needed it, but I agree, when the app is running it is better without.